server/src/main/java/org/candlepin/resource/StatusResource.java

```diff
@@ -83,7 +84,14 @@ public StatusResource(RulesCurator rulesCurator, Configuration config, JsRunnerP
         if (config == null || !config.getBoolean(ConfigProperties.STANDALONE)) {
             standalone = false;
         }
+
         adapterConfig = keycloakAdapterConfiguration.getAdapterConfig();
+        if (adapterConfig != null && config != null &&
+            config.getBoolean(ConfigProperties.KEYCLOAK_AUTHENTICATION)) {
```

Comment:
I think the comment went missing due to this line changing, as I'm unable to respond directly to it; but generally speaking when it comes to these types of checks, I tend to be very defensive and default to something that won't fail in a way that throws unexpected exceptions if we can avoid it. In this particular case, we're relying on the default being set outside of this object's influence which is something I tend to cover.

Not a huge deal, but in some sense a stylistic thing.